src/main/kotlin/no/nav/syfo/dialogmote/api/v2/DialogmoteApiV2.kt

```diff
@@ -20,6 +21,8 @@ const val dialogmoteApiV2Basepath = "/api/v2/dialogmote"
 
 const val dialogmoteApiPersonIdentUrlPath = "/personident"
 
+const val dialogmoteApiMoterUrlPath = "/moter"
```

Comment:
Endepunket kan heller hete f.eks `/veilderident` for å illustrere at det er en liste med DialogmoteDTO for veilederident, i motsetning til `/personident` som er det andre endepunket her.

Reply:
Men endepunktet returnerer ikke en `veilederident` da, så da gir det vel egentlig ikke mening at pathen ender på "veilederident"?

Reply:
@JMLindseth Forsåvidt enig i det. Hvis det skulle vært med mulig rest-likt så kunne det f.eks vært veilederident/dialogmote. Så nå er bare dialogmote før veilederident og ikke etter, 

Men det er i hvert fall bedre med  `/api/v2/dialogmote/veilederident` enn "/api/v2/dialogmote/moter".

Reply:
Tja, man trenger vel ikke "veilederident" i pathen i det hele tatt? Url'en skal jo referere til et objekt, litt ala mappestruktur. Veilederident skulle vel i så fall vært en path param, hvis det hadde vært lov. Evt. bare bruke `veileder/dialogmote`, fordi en veileder kan ha møter på seg, mens en ident ikke er et objekt.

Det er jo helt innafor å bare ha `dialogmote/moter` her: Man skal jo hente ressursen "møter", og så spesifiserer man lenger inn i urlen. Ofte gjøres det med query/path params, men vi bruker headers i stedet.

Reply:
Om man kaller det `veileder` eller `veilederident`, blir potato potato for min del, så greit det.

Ikke enig i det siste du skriver.(`Det er jo helt innafor å bare ha dialogmote/moter`)
I streng REST-kontekst så gir ikke `dialogmote/moter` noe mening. 
Først og fremst fordi et mote==dialogmote. For det andre så bruker man ikke flertall for å hente liste i resful, men man bruker entall for å hente liste(`../dialogmote`). Det sier heller ikke noe om hvilke møter man henter.